src/mon/MonMap.h

```diff
@@ -135,9 +137,10 @@ class MonMap {
   uint8_t min_mon_release = 0;
 
   void _add_ambiguous_addr(const string& name,
-			   entity_addr_t addr,
-			   int priority,
-			   bool for_mkfs=false);
+                           entity_addr_t addr,
+                           int priority = 0,
+                           int weight = 0,
```

Comment:
i'd suggest avoid using the default parameters as much as possible here. as it's a private method. it'd be better if we can be more explicit.